### PR TITLE
[Forwardport] Fixed set template syntax in block file

### PR DESCRIPTION
**Pull request by gelanivishal**
### Original Pull Request
https://github.com/magento/magento2/pull/15339

### Description

Fixed set template syntax in block file

### Fixed Issues (if relevant)

1. magento/magento2#15345:Template syntax in block file
### Manual testing scenarios

N/A

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
Hi @gelanivishal. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me {$VERSION} instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://github.com/magento/magento2/wiki/Magento-Contributor-Assistant) documentation

File: app/code/Magento/Paypal/Block/Iframe.php [hunk 1]
```diff
@@ -116,7 +116,7 @@ protected function _construct()
             if ($file && $directory->isExist($directory->getRelativePath($file))) {
                 $this->setTemplate($templateFile);
             } else {
-                $this->setTemplate('hss/iframe.phtml');
+                $this->setTemplate('Magento_Paypal::hss/iframe.phtml');
```
Comment (sidolov) [inline app/code/Magento/Paypal/Block/Iframe.php]: Please, revert fix in places when the template specified dynamically

File: app/code/Magento/Paypal/Block/Iframe.php [hunk 2]
```diff
@@ -198,7 +198,7 @@ protected function _beforeToHtml()
     protected function _toHtml()
     {
         if ($this->_isAfterPaymentSave()) {
-            $this->setTemplate('hss/js.phtml');
+            $this->setTemplate('Magento_Paypal::hss/js.phtml');
```
Comment (sidolov) [inline app/code/Magento/Paypal/Block/Iframe.php]: Please, revert fix in places when the template specified dynamically

File: app/code/Magento/Review/Block/Form.php [hunk 3]
```diff
@@ -139,7 +139,7 @@ protected function _construct()
             );
         }
 
-        $this->setTemplate('form.phtml');
+        $this->setTemplate('Magento_Review::form.phtml');
```
Comment (sidolov) [inline app/code/Magento/Review/Block/Form.php]: Please, revert fix in places when the template specified dynamically

File: app/code/Magento/Review/Block/Rating/Entity/Detailed.php [hunk 4]
```diff
@@ -49,7 +49,7 @@ protected function _toHtml()
         $reviewsCount = $this->_ratingFactory->create()->getTotalReviews($entityId, true);
         if ($reviewsCount == 0) {
             #return __('Be the first to review this product');
-            $this->setTemplate('empty.phtml');
+            $this->setTemplate('Magento_Review::empty.phtml');
```
Comment (sidolov) [inline app/code/Magento/Review/Block/Rating/Entity/Detailed.php]: Please, revert fix in places when the template specified dynamically

File: app/code/Magento/Tax/Plugin/Checkout/CustomerData/Cart.php [hunk 5]
```diff
@@ -66,7 +66,7 @@ public function afterGetSectionData(\Magento\Checkout\CustomerData\Cart $subject
             foreach ($result['items'] as $key => $itemAsArray) {
                 if ($item = $this->findItemById($itemAsArray['item_id'], $items)) {
                     $this->itemPriceRenderer->setItem($item);
-                    $this->itemPriceRenderer->setTemplate('checkout/cart/item/price/sidebar.phtml');
+                    $this->itemPriceRenderer->setTemplate('Magento_Tax::checkout/cart/item/price/sidebar.phtml');
```
Comment (sidolov) [inline app/code/Magento/Tax/Plugin/Checkout/CustomerData/Cart.php]: Please, revert fix in places when the template specified dynamically

File: app/code/Magento/UrlRewrite/Block/Edit.php [hunk 6]
```diff
@@ -65,7 +65,7 @@ public function __construct(
      */
     protected function _prepareLayout()
     {
-        $this->setTemplate('edit.phtml');
+        $this->setTemplate('Magento_UrlRewrite::edit.phtml');
```
Comment (sidolov) [inline app/code/Magento/UrlRewrite/Block/Edit.php]: Please, revert fix in places when the template specified dynamically

File: app/code/Magento/User/Block/Role/Tab/Users.php [hunk 7]
```diff
@@ -45,7 +45,9 @@ protected function _construct()
         $roleId = $this->getRequest()->getParam('rid', false);
         /** @var \Magento\User\Model\ResourceModel\User\Collection $users */
         $users = $this->_userCollectionFactory->create()->load();
-        $this->setTemplate('role/users.phtml')->assign('users', $users->getItems())->assign('roleId', $roleId);
+        $this->setTemplate('Magento_User::role/users.phtml')
```
Comment (sidolov) [inline app/code/Magento/User/Block/Role/Tab/Users.php]: Please, revert fix in places when the template specified dynamically

**Review by sidolov — CHANGES_REQUESTED** [review]

File: app/code/Magento/Paypal/Block/Iframe.php (hunk 1 above)
Comment (gelanivishal) [reply app/code/Magento/Paypal/Block/Iframe.php]: Done.

File: app/code/Magento/Paypal/Block/Iframe.php (hunk 2 above)
Comment (gelanivishal) [reply app/code/Magento/Paypal/Block/Iframe.php]: Done.

File: app/code/Magento/Review/Block/Form.php (hunk 3 above)
Comment (gelanivishal) [reply app/code/Magento/Review/Block/Form.php]: Done.

File: app/code/Magento/Review/Block/Rating/Entity/Detailed.php (hunk 4 above)
Comment (gelanivishal) [reply app/code/Magento/Review/Block/Rating/Entity/Detailed.php]: Done.

File: app/code/Magento/Tax/Plugin/Checkout/CustomerData/Cart.php (hunk 5 above)
Comment (gelanivishal) [reply app/code/Magento/Tax/Plugin/Checkout/CustomerData/Cart.php]: Done.

File: app/code/Magento/UrlRewrite/Block/Edit.php (hunk 6 above)
Comment (gelanivishal) [reply app/code/Magento/UrlRewrite/Block/Edit.php]: Done.

File: app/code/Magento/User/Block/Role/Tab/Users.php (hunk 7 above)
Comment (gelanivishal) [reply app/code/Magento/User/Block/Role/Tab/Users.php]: Done.

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]
Hi,
It looks like you applied not all changes.
Take a look at 46e3c0c228d887a8d51c04fac68c9415109ad00d - forwardport should contains the same changeset

**Comment by gelanivishal** [comment]
Hello @ihor-sviziev,

Please review this commit b405dfaddc12ff42f90425de1f45530cced78327 and also check @sidolov's requested comments.

Thank you.

**Comment by ihor-sviziev** [comment]
@gelanivishal sure, you reverted changes that @sidolov requested, but it's still not full list of changes.
Example: there are changes in magento newsletter, offline payments modules, but in this PR there are no such changes. Just compare 46e3c0c with your PR

**Comment by gelanivishal** [comment]
Hello @ihor-sviziev, @sidolov,

Other files are already changes in 2.3-develop, Please check your side at once. 

Thank you.

**Review by ihor-sviziev — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @ihor-sviziev, thank you for the review.
[ENGCOM-2593](https://jira.corp.magento.com/browse/ENGCOM-2593) has been created to process this Pull Request

**Comment by magento-engcom-team** [comment]
Hi @gelanivishal. Thank you for your contribution. 
We will aim to release these changes as part of 2.3.0. 
Please check the release notes for final confirmation.

**Pull request merged**